msu/systems/serialization/serialization_system.nut

```diff
@@ -14,4 +16,56 @@
 		this.Mods.push(_mod);
 		_mod.Serialization = ::MSU.Class.SerializationModAddon(_mod);
 	}
+
+	function flagSerialize( _mod, _id, _object, _flags = null, _clear = true )
+	{
+		if (_flags == null) _flags = ::World.Flags;
+		local outEmulator = ::MSU.Class.SerializationEmulator(_mod, _id);
+		::MSU.Utils.serialize(_object, outEmulator);
+		outEmulator.storeInFlagContainer(_flags);
+		if (_clear) this.FlagsToClear.push([outEmulator.getEmulatorString(), _flags]);
```

Comment:
Is the _clear optional argument necessary? Shouldn't it always clear?

Reply:
Same question as Taro above.